packages/inference/src/snippets/getInferenceSnippets.ts

```diff
@@ -167,7 +167,9 @@ const snippetGenerator = (templateName: string, inputPreparationFn?: InputPrepar
 		const accessTokenOrPlaceholder = opts?.accessToken ?? placeholder;
 
 		/// Prepare inputs + make request
-		const inputs = inputPreparationFn ? inputPreparationFn(model, opts) : { inputs: getModelInputSnippet(model) };
+		const inputs = inputPreparationFn
+			? inputPreparationFn(model, opts)
```

Comment:
I guess you also need to add support for this in the different `inputPreparationFn`s?